modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,425 @@
+= Configure GCP Private Service Connect for BYOC
+:description: Set up GCP Private Service Connect to securely access Redpanda Cloud. 
+:page-cloud: true
+
+include::shared:partial$feature-flag.adoc[]
+
+// This guide describes the steps for setting up Private Service Connect (PSC) on Redpanda Cloud.
+
+// Redpanda GCP PSC can be enabled during cluster creation or for an existing cluster.
+
+// After completing this guide, a PSC service attachment will have been deployed for your Redpanda Cloud cluster. You may then establish connections to it from any GCP project included in the allow list.
+
```

Comment:
```suggestion
Redpanda GCP Private Service Connect (PSC) provides secure access to Redpanda Cloud from your own VPC. Traffic over PSC does not go through the public internet because a PSC connection is treated as its own private GCP service. While your VPC has access to the Redpanda VPC, Redpanda cannot access your VPC.

Consider using PSC if you have multiple VPCs and could benefit from a more simplified approach to network management:

* PSC allows overlapping CIDR ranges in VPC networks.
* PSC does not limit the number of connections.
* You control from which GCP projects connections are allowed.
```